src/Agent/NewRelic/Agent/Core/Configuration/ReportedConfiguration.cs

```diff
@@ -742,6 +742,10 @@ public IReadOnlyDictionary<string, string> GetAppSettings()
         [JsonProperty("agent_control.health.frequency")]
         public int HealthFrequency => _configuration.HealthFrequency;
 
+        [JsonIgnore]
```

Comment:
I think this is a configuration setting we would want to report to New Relic along with the rest of agent config, to help with support troubleshooting.

```suggestion
        [JsonProperty("agent.aws_lambda_apm_mode_enabled")]
```

Reply:
There's no connect cycle in serverless mode, so it doesn't matter.